packages/go_router/pubspec.yaml

```diff
@@ -22,6 +22,7 @@ dev_dependencies:
   flutter_test:
     sdk: flutter
   io: ^1.0.4
+  leak_tracker_flutter_testing: any
```

Comment:
can you pin a version instead of any?

Reply:
Actually, the documentation recommends not to pin any number: see https://github.com/dart-lang/leak_tracker/blob/main/doc%2Fleak_tracking%2FDETECT.md

Do you still want me to pin one after reading this?

Reply:
oh i wasn't aware that is the guideline, thanks for pointing it out. LGTM

Reply:
It looks like the pipeline `Linux repo_checks` is not happy with this. Should it be ignored (and how?) ? 
Or is there something else we can do about it?

Reply:
I think we should ignore it given that is how leak tracking suggests.

I think you need to modify packages/script/configs/allowed_unpinned_deps.yaml

cc @stuartmorgan

Reply:
> I think you need to modify packages/script/configs/allowed_unpinned_deps.yaml

It looks like the tooling doesn't check for `any` specifically, which it should, so that should work. (At some point I'll need to fix that and split this exception out into its own file, since in general we don't want to allow `any`.)